package.json

```diff
@@ -19,30 +19,40 @@
   },
   "author": "Titus Wormer <tituswormer@gmail.com> (https://wooorm.com)",
   "contributors": [
-    "Titus Wormer <tituswormer@gmail.com> (https://wooorm.com)"
+    "Titus Wormer <tituswormer@gmail.com> (https://wooorm.com)",
+    "Lucas Brandstaetter <lucas@brandstaetter.tech> (https://github.com/Roang-zero1)"
   ],
+  "files": [
+    "index.js",
+    "index.d.ts"
+  ],
+  "types": "index.d.ts",
   "dependencies": {
     "unist-util-is": "^4.0.0"
   },
   "devDependencies": {
+    "@types/tape": "^4.2.33",
```

Comment:
Where are these used?

Reply:
I use typescript to verify the JavaScript code.
In order to correctly verify the tests I had to add the types for the tape library.